modules/seqexec/server/src/main/scala/seqexec/server/nifs/NifsEpics.scala

```diff
@@ -3,15 +3,19 @@
 
 package seqexec.server.nifs
 
-import java.lang.{Double => JDouble}
-
+import cats.data.Nested
+import cats.effect.IO
+import cats.effect.Sync
+import cats.implicits._
 import edu.gemini.epics.acm._
 import edu.gemini.seqexec.server.nifs.DhsConnected
+import java.lang.{Double => JDouble}
 import org.log4s.{Logger, getLogger}
 import seqexec.server.EpicsCommand.setParameter
 import seqexec.server.{EpicsCommand, EpicsSystem, ObserveCommand, SeqAction}
+import seqexec.server.EpicsUtil.safeAttribute
 
-class NifsEpics(epicsService: CaService, tops: Map[String, String]) {
+class NifsEpics[F[_]: Sync](epicsService: CaService, tops: Map[String, String]) {
   val NifsTop = tops.getOrElse("niri", "niri:")
```

Comment:
Is this correct that `NifsTop` should use `niri`?

Reply:
Doh' good catch